rules/aws_cloudtrail_rules/aws_console_login_without_mfa.py

```diff
@@ -42,7 +42,7 @@ def rule(event):
     new_user_string = (
         event.deep_get("userIdentity", "userName", default="<MISSING_USER_NAME>")
         + "-"
-        + event.deep_get("userIdentity", "principalId", default="<MISSING_ID>")
+        + event.udm("actor_user")
```

Comment:
my unit tests are breaking now because `event.udm("actor_user")` evaluates to `None`. how does this value get supplied in unit tests?